ext/opentelemetry-ext-otcollector/src/opentelemetry/ext/otcollector/metrics_exporter/__init__.py

```diff
@@ -145,11 +145,12 @@ def get_collector_metric_type(metric: Metric) -> metrics_pb2.MetricDescriptor:
 
 
 def get_collector_point(metric_record: MetricRecord) -> metrics_pb2.Point:
+    # TODO: horrible hack to get original list of keys to then get the bound
```

Comment:
Why is this a horrible hack? :thinking:

Reply:
Because I have a list of keys ready to be used as key in a map and I have to convert they back to a normal map (not usable as key) to pass them to the `bind()` function where they are going to be converted (again) in a usable dict key.

Please notice that this logic will be removed soon https://github.com/open-telemetry/opentelemetry-python/pull/522/files#r397198165.